optuna/storages/_base.py

```diff
@@ -17,8 +18,11 @@
 from optuna.trial import TrialState
 
 
+_CONSTRAINTS_KEY = "constraints"
```

Comment:
I would like to use `_CONSTRAINT_KEY` in `optuna.sampler._base`, but it would be a circular import, so it is a direct value. Please let me know if there is a better way.